Build/scripts/vso_build_wrapper.sh

```diff
@@ -0,0 +1,41 @@
+#-------------------------------------------------------------------------------------------------------
+# Copyright (C) Microsoft. All rights reserved.
+# Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+#-------------------------------------------------------------------------------------------------------
+
+BUILD_COMMANDLINE=""
+
+while [[ $# -gt 0 ]]; do
+    case "$1" in
+    --debug)
+        BUILD_COMMANDLINE="-d"
+        ;;
+    --test)
+        BUILD_COMMANDLINE="-t"
+        ;;
+    --release)
+        BUILD_COMMANDLINE=""
+        ;;
+    --x64)
+        BUILD_ARCH="--x64"
+        ;;
+    --x86)
+        BUILD_ARCH="--x86"
+        ;;
+    --arm)
+        BUILD_ARCH="--arm"
+        ;;
+    esac
+
+    shift
+done
+
```

Comment:
Shouldn't logic for overriding params be included in the .build.sh script to begin with? the parameter parsing here seems redundant (except to print the echo lines below, which could be moved into build.sh itself).